markdown/usage/institution.md

```diff
@@ -0,0 +1,44 @@
+
+## Adding your institution to the contributor's list
+
+
+Haven't you find your organization / group of contributors on the [community pages](https://nf-co.re/community)? Please add yourself! Just few lines of a YAML file needed.
```

Comment:
```suggestion
In case you couldn't find your organization / group of contributors on the [community pages](https://nf-co.re/community), please add yourself! Just a few lines of YAML and some icon / logos needed.
```